### ISSUE TITLE
Improve the definition of the path of og:image  - take from api

### ISSUE THREAD

**Issue by eliselavy** [C2DH/journal-of-digital-history#646]
Now we have. hardcoded value in:

https://github.com/C2DH/journal-of-digital-history/blob/develop/public/index.html

with several issue running in parallel how to do it?

Social media cover for issue are here:

https://github.com/C2DH/journal-of-digital-history/tree/develop/public/img/issues/social-media

to take also from api - raw ul

**Comment by danieleguido** [comment]
the og:image in the index.html file is the image that appears when referring to the root url, so it can be changed manually according to the editorial needs: call for papers, new issue, etc.. as it was done in [v2.4.6](https://github.com/C2DH/journal-of-digital-history/commit/434c01d20ed25b52a70fa0820a0f3458f60a0203)

**Comment by danieleguido** [comment]
still todo (I put low priority for now): 
- create a helmet for the issue (similar to `src/components/ArticleV2/ArticleHelmet`)
- load the helmet when `src/pages/Articles` got the issue id params (i.e. when url is https://journalofdigitalhistory.org/en/issue/jdh002) see routing rule in App component: https://github.com/C2DH/journal-of-digital-history/blob/f854ce1bcc91baa72b61a9125d68ef65fa9c1ccb/src/App.js#L149
-
